zerver/management/commands/export.py

```diff
@@ -116,9 +124,11 @@ def handle(self, *args: Any, **options: Any) -> None:
         num_threads = int(options['threads'])
         if num_threads < 1:
             raise CommandError('You must have at least one thread.')
-
+        if public_only and consent_message_id is not None:
+            raise CommandError('Please pass either --public-only or --consent-message-id')
```

Comment:
Let's not do this validation -- I think there's valid use cases for doing neither (e.g. exporting the data from your own Zulip server).  

Just to provide some context, longer-term we're likely to integrate this with #11996 to provide a completely automated flow.

Reply:
hmm. Actually I meant to say don't pass both `--public-only` or `--consent-message-id`.